packages/js/product-editor/src/components/tabs/tabs.tsx

```diff
@@ -84,7 +84,31 @@ export function Tabs( { onChange = () => {} }: TabsProps ) {
 		_childIndex: number,
 		child: HTMLButtonElement
 	) {
-		child.click();
+		child.focus();
+	}
+
+	function handleKeyDown( event: KeyboardEvent< HTMLDivElement > ) {
+		const tabs =
+			event.currentTarget.querySelectorAll< HTMLButtonElement >(
+				'[role="tab"]'
+			);
+
+		switch ( event.key ) {
+			case 'Home':
+				event.preventDefault();
+				event.stopPropagation();
+
+				const [ firstTab ] = tabs;
+				firstTab?.focus();
+				break;
+			case 'End':
+				event.preventDefault();
+				event.stopPropagation();
+
+				const [ lastTab ] = [ ...tabs ].reverse();
```

Comment:
A funny way to get the last tab instead of just doing: `tabs[ tabs.length - 1 ]`

Reply:
True, I guess I was unnecessary creative here!